corehq/apps/app_manager/migrations/0009_add_sqlglobalappconfig.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-11-22 18:53
+
+from django.core.management import call_command
+from django.db import migrations, models
+
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _populate_sql_global_app_config(apps, schema_editor):
+    call_command('populate_sql_global_app_config')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app_manager', '0008_remove_uses_master_app_form_ids'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SQLGlobalAppConfig',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('domain', models.CharField(max_length=255)),
+                ('app_id', models.CharField(max_length=255)),
+                ('app_prompt', models.CharField(default='off', max_length=32, null=True,
+                                                choices=[('on', 'on'), ('off', 'off'), ('forced', 'forced')])),
+                ('apk_prompt', models.CharField(default='off', max_length=32, null=True,
+                                                choices=[('on', 'on'), ('off', 'off'), ('forced', 'forced')])),
+                ('apk_version', models.CharField(max_length=32, null=True)),
+                ('app_version', models.IntegerField(null=True)),
+            ],
+        ),
+        migrations.AlterUniqueTogether(
+            name='sqlglobalappconfig',
+            unique_together=set([('domain', 'app_id')]),
+        ),
+        migrations.RunPython(_populate_sql_global_app_config,
```

Comment:
should this be run outside of the migration? in your follow up PR, you have the check to ensure that it has been run, so I think it would be ideal to be able to run this manually on large environments with lots of apps instead of during deploy

Reply:
This is a good point for this general process. I don't think it applies to this specific migration, as these docs are tiny and there are only 31 of them on prod, 7 on india.